app/components/forms/events/view/create-access-code.js

```diff
@@ -102,7 +102,7 @@ export default Component.extend(FormMixin, {
   accessLink : computed('data.code', function() {
     const params = this.get('router._router.currentState.routerJsState.params');
     const origin = this.get('fastboot.isFastBoot') ? `${this.get('fastboot.request.protocol')}//${this.get('fastboot.request.host')}` : location.origin;
-    let link = origin + this.router.urlFor('public', params['events.view'].event_id, { queryParams: { access_code: this.get('data.code') } });
+    let link = origin + this.router.urlFor('public', params['events.view'].event_id, { queryParams: { code: this.get('data.code') } });
```

Comment:
I guess we are using `code` for both `discount code` as well .

Reply:
![Screenshot from 2020-01-21 21-39-26](https://user-images.githubusercontent.com/46647141/72821458-89083980-3c96-11ea-92e0-887cb76d4a44.png)